libcxx/test/libcxx/transitive_includes/cxx23.csv

```diff
@@ -86,7 +86,6 @@ chrono sstream
 chrono stdexcept
 chrono string
 chrono string_view
-chrono tuple
```

Comment:
AFAIK we wanted to keep our transitive set stable. Maybe we should discuss this policy during a monthly meeting. Based on the several remarks on the size by the Chromium team and recently #80196 I think we should revise our header stability policy or at least have a plan how to remove transitive includes.

@vitaut  recently [blogged](https://vitaut.net/posts/2024/faster-cpp-compile-times/) about this too, and `_LIBCPP_REMOVE_TRANSITIVE_INCLUDES` does not seem to be well known.

@ldionne can you put this on the agenda?

Reply:
> Until now we've only kept the includes up to C++20 stable. C++23 was always fair game. I agree that we should rethink our policy on this, since the amount of transitive includes that we could avoid has become massive.

I know, but at some point it was said only the C++ development version is fair game, which would mean C++26. Hence the proposal to discuss this. I was already considering to put it on the agenda after a private conversation with Victor.


> I'm currently also working on a patch to reduce the transitive includes further by including things in the top level headers by version.

The last time I had a patch to improve the transitive includes by `#ifdef-ing` it got a lukewarm response. So nice to hear the response now is better.

Reply:
Based on the last monthly meeting we didn't declare C++23 includes frozen.